apps/react/src/models/genres-filter-params.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseFilterParams } from './base-filter-params';
+
+/** Genres filter params. */
+export namespace GenresFilterParams {
+
+	/** Ordering. */
+	export type Ordering = Readonly<{
+
+		/** Ordering. */
+		ordering: string | null;
+	}>;
+
+	/** Filter type. */
+	export enum FilterType {
+		Genres = 'GENRES',
+		ExplicitGenres = 'EXPLICIT_GENRES',
+		Themes = 'THEMES',
+		Demographics = 'DEMOGRAPHICS',
+	}
+
+	/** Type. */
+	export type Type = Readonly<{
+
+		/** Type. */
+		type: string | null;
+	}>;
+
+	/** Combined. */
+	export type Combined = BaseFilterParams.Combined & Ordering & Type;
+}
```

Comment:
It seems you are not using `GenreFilterParams.Type` and `GenreFilterParams.Ordering`. Then why complicate it?

```suggestion
/** Filter type. */

export enum GenreFilterType {
  Genres = 'GENRES',
  ExplicitGenres = 'EXPLICIT_GENRES',
  Themes = 'THEMES',
  Demographics = 'DEMOGRAPHICS',
}
export type GenreFilterParams =  BaseFilterParams & {
    readonly ordering: string;
    readonly type: GenreFilterType;
}
```

Reply:
We have FilterType both in QueryParams and FilterParams, so in my opinion it's good to put them in a namespace, otherwise, we have to deal with some pretty long names like GenreFilterTypeQueryParams and GenreFilterTypeFilterParams to distinguish between 2 GenresFilterType. What is your opinion about this, bratan?